specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/payments.json

```diff
@@ -0,0 +1,711 @@
+          "description": "The account holder name for the payment method. This is only supported for payment methods with family CreditCard.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "displayName": {
+          "description": "The display name of the payment method.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "expiration": {
+          "description": "The expiration month and year of the payment method. This is only supported for payment methods with family CreditCard.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "family": {
+          "description": "The family of payment method.",
+          "enum": [
+            "CreditCard",
+            "CheckWire"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "paymentMethodFamily"
+          }
+        },
+        "lastFourDigits": {
+          "description": "Last four digits of payment method.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "logos": {
+          "description": "The list of logos for the payment method.",
+          "items": {
+            "$ref": "#/definitions/PaymentMethodLogo"
+          },
+          "type": "array"
+        },
+        "status": {
+          "description": "Status of the payment method.",
+          "enum": [
+            "active",
+            "inactive"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "paymentMethodStatus"
+          }
+        },
+        "type": {
+          "description": "The type of payment method.",
+          "readOnly": true,
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "PaymentMethodResource": {
+      "description": "The payment method resource model definition.",
+      "properties": {
+        "paymentMethodId": {
+          "description": "The ID that uniquely identifies a payment method.",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "PaymentMethodsListResult": {
+      "description": "The list of payment methods.",
+      "properties": {
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "value": {
+          "description": "The list of payment methods.",
+          "items": {
+            "$ref": "#/definitions/PaymentMethod"
+          },
+          "readOnly": true,
+          "type": "array"
+        }
+      },
+      "type": "object"
+    }
+  },
+  "host": "management.azure.com",
+  "info": {
+    "description": "The billing client allows you to view and manage your billing details programmatically.",
+    "title": "PaymentManagementClient",
+    "version": "2023-04-01"
+  },
+  "parameters": {
+    "apiVersionParameter": {
```

Comment:
Can `apiVersionParameter`, `billingAccountName`, `billingProfileName` all refer to types.json?

Also, I believe ARM swagger validation will ask you to supply a regex for paymentMethodName since it is a resource name.